presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix query exceeding local memory limit when unspilling data due to incorrect memory size calculation.
+* Fix deadlock for queries with join and limit when spilling is enabled.
+* Fix expressions with multiple ``IS NULL`` on the same variable.
+* Improve memory usage of queries spilling in the join operator.
+* Improve plan expressions to be closer to valid SQL.
```

Comment:
`plan expressions` makes me think that this is some kind of an expression. So lets clarify that we are talking about expressions printed in plans.

```
Improve expressions printed in query plans to be closer to valid SQL.
```

Reply:
done